### PR TITLE
Crud companyvalue

### PR DESCRIPTION
**Pull request by kajawitek**

### CODE REVIEW

File: app/controllers/admins/company_values_controller.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    def new
+      @company_value = CompanyValue.new
+    end
+
+    def create
+      company_value_duplicate = CompanyValue.find_by(title: company_value_params[:title])
+
+      if company_value_duplicate.present?
+        redirect_to new_admins_company_value_url, notice: 'Title must be unique.'
+        # render json: {errors: {message: 'The same uuid already exists.'}}, status: :accepted
```
Comment (rniedziolka) [inline app/controllers/admins/company_values_controller.rb:24]: Unnecessary comment :)

File: app/controllers/admins/company_values_controller.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      end
+
+      @company_value = CompanyValue.new(company_value_params)
+
+      if @company_value.title.empty?
+        redirect_to new_admins_company_value_url, notice: 'Title is empty. Company Value was not saved.'
+      else
+        # if params[:company_value][:title].exists?
+        #   redirect_to new_admins_company_value_url, notice: 'Title must be unique.'
+        # else
```
Comment (rniedziolka) [inline app/controllers/admins/company_values_controller.rb:35]: comments generally make the code hard to read, I would get rid of them altogether

File: db/seeds.rb [hunk 3]
```diff
@@ -16,4 +16,10 @@
 
 Admin.where(email: "admin@test.com").first_or_create!(password: "Password1")
 
+CompanyValue.where(title: "Honesty").first_or_create!
+CompanyValue.where(title: "Ownership").first_or_create!
+CompanyValue.where(title: "Accountability").first_or_create!
+CompanyValue.where(title: "Passion").first_or_create!
+
+
```
Comment (rniedziolka) [inline db/seeds.rb:25]: No newline at the end of file :)

File: spec/features/admin_company_value_spec.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    click_link 'Back'
+
+    expect(page).to have_content 'Company Values Title'
+
+    click_link 'Edit'
+
+    fill_in 'company_value[title]', with: 'changed_title'
+    click_button 'Update Company value'
+
+    expect(page).to have_content 'Company Value was successfully updated.'
```
Comment (rniedziolka) [inline spec/features/admin_company_value_spec.rb:42]: Would be good to check, if the title has been changed.

File: spec/features/admin_company_value_spec.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+    click_link 'Destroy'
+
+    expect(page).to have_content 'Company Value was successfully destroyed.'
```
Comment (rniedziolka) [inline spec/features/admin_company_value_spec.rb:46]: And check, that title is no longer visible on the page :)

File: spec/features/admin_company_value_spec.rb [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+RSpec.describe 'Admin employees spec', type: :feature do
+  let!(:admin) { create(:admin) }
+  let!(:company_value) { build(:company_value) }
+
+  it 'CRUD company value' do
+    visit admins_root_path
+
+    fill_in 'admin[email]', with: admin.email
+    fill_in 'admin[password]', with: admin.password
+    click_button 'Log in'
```
Comment (rniedziolka) [inline spec/features/admin_company_value_spec.rb:12]: Consider using devise login helper here - you have already covered user logging with tests, so there is not necessary to check it here again.

https://localcoder.org/how-to-access-devise-current-user-in-a-rspec-feature-test

**Comment by rniedziolka** [comment]
Good job Kaja! ❤️‍🔥 
Most of my comments are cosmetic, but let me know if you have any questions 😄 
I would recommend reading about naming conventions for commits and branches:
https://dev.to/couchcamote/git-branching-name-convention-cch

File: app/models/company_value.rb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+class CompanyValue < ApplicationRecord
+  validates :title, presence: true
```
Comment (oskarlakner) [inline app/models/company_value.rb:2]: you could combine these two validations in a one-liner

**Pull request merged**